csv_wrangler/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.1.3'
+__version__ = '2.0.0'
```

Comment:
We should just bump this to `1.0.0`, no need to do 2 major version bumps

Reply:
Woops, you're right